package-res/resources/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -89,16 +89,21 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
           var parameterValue = this.dashboard.getParameterValue(this.parameter),
               value = undefined;
 
-          if(parameterValue.length == 1) {
+          // If no value was specified use current date
+          if (!parameterValue.length){
```

Comment:
parameterValue wil return an empty array if no value is specified when creating the prompt, so I have to check for its length.